core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -86,8 +86,13 @@ private[spark] class EventLoggingListener(
    * Creates the log file in the configured log directory.
    */
   def start() {
-    if (!fileSystem.isDirectory(new Path(logBaseDir))) {
-      throw new IllegalArgumentException(s"Log directory $logBaseDir does not exist.")
+    try{
+        if( !fileSystem.getFileStatus( new Path( logBaseDir ) ).isDir() ){
```

Comment:
This is still not right. Just take a look at the code Josh posted, or the code you're replacing. Basically, do this:

    if (foo) {

Not this:

    if( foo ){